### PR TITLE
Add "Web NFC" to "New capabilities status" page

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR simply adds "Web NFC" to the new "New capabilities status" page and updates the Web NFC article to link back to it.

### CODE REVIEW

File: src/site/content/en/blog/nfc/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Web apps should be able to do anything native apps can. The [Capabilities
 project](https://developers.google.com/web/updates/capabilities), of which Web
```
Comment (petele) [inline src/site/content/en/blog/nfc/index.md]:
```suggestion
project](/fugu-status/), of which Web
```

File: src/site/content/en/blog/fugu-status/index.md [hunk 2]
```diff
@@ -155,6 +155,15 @@ trials in the [Origin Trials Guide for Web Developers][ot-guide].
           running.
         </td>
       </tr>
+      <tr>
+        <td>
+          <a href="/nfc/">Web NFC</a>
+        </td>
+        <td>
+          Web NFC provides sites the ability to read and write to NFC tags when
+          they are in close proximity to the user's device.
```
Comment (petele) [inline src/site/content/en/blog/fugu-status/index.md]: Can we add a use case example here as well.

> For example, museums and art galleries could display additional information about a display when the user touches their device to an NFC card near the exhibit; or an inventory management web app could read or write data to sn NFC tag on a container to update information on its contents.

File: src/site/content/en/blog/nfc/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md]: Done

File: src/site/content/en/blog/fugu-status/index.md (hunk 2 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/fugu-status/index.md]: Done

**Comment by beaufortfrancois** [comment]
@petele Your feedback has been addressed.

File: src/site/content/en/blog/fugu-status/index.md [hunk 3]
```diff
@@ -155,6 +155,20 @@ trials in the [Origin Trials Guide for Web Developers][ot-guide].
...
+          For example, museums and art galleries could display additional
+          information about a display when the user touches their device to an
+          NFC card near the exhibit; or an inventory management web app could
+          read or write data to sn NFC tag on a container to update information
```
Comment (kaycebasques) [inline src/site/content/en/blog/fugu-status/index.md]:
```suggestion
          read or write data to an NFC tag on a container to update information
```

**Comment by kaycebasques** [comment]
Confirmed that you already have access to push branches to googlechrome/web.dev

**Comment by kaycebasques** [comment]
Ah, nevermind, I see that you gave me edit access to your fork. I wish the GitHub UI made that more clear!

**Pull request merged**